conformance/client/google-java/build.gradle.kts

```diff
@@ -0,0 +1,55 @@
+buildscript {
+    repositories {
+        mavenCentral()
+        gradlePluginPortal()
+    }
+    dependencies {
+        classpath("gradle.plugin.com.github.johnrengelman:shadow:7.1.2")
```

Comment:
If possible we should try to keep all the versions defined in the gradle toml version catalogs.